projects/Mallard/android/app/build.gradle

```diff
@@ -92,6 +92,7 @@ def enableSeparateBuildPerCPUArchitecture = false
  * Run Proguard to shrink the Java bytecode in release builds.
  */
 def enableProguardInReleaseBuilds = false
+def vcode = (int)(((new Date().getTime()/1000) - 1451606400) / 10)
```

Comment:
It is probably better to use the HEAD commit hash so builds are reproducible and unique based on code as well as being easier to track